packages/bull-vault/src/BullStrategy.sol

```diff
@@ -140,7 +143,7 @@ contract BullStrategy is ERC20, LeverageBull {
     }
 
     /**
-     * @notice deposit to crab: deposits crab and ETH, receives USDC and wPowerPerp
+     * @notice deposit to crab: deposits crab and ETH, receives USDC, wPowerPerp and Bull token
```

Comment:
Controversial lack of oxford comma

"The wombat eats shoots, roots, and leaves."

Reply:
I'm a casual comma user, non of that snobby comma stuff